ddtrace/propagation/http.py

```diff
@@ -1138,7 +1128,7 @@ def my_controller(url, headers):
                     context = propagator._extract(normalized_headers)  # type: ignore
                     if config.propagation_http_baggage_enabled is True:
```

Comment:
Doe we need to make any changes to this behavior? How does your adding of baggage intersect with this?

Reply:
I think there should be minimal interaction. If `config.propagation_http_baggage_enabled` is True, then the values will be merged into `_baggage`. The headers are different so they can be propagated separately. The idea is to not touch `config.propagation_http_baggage_enabled` because it would lead to a breaking change (RFC).